model_compression_toolkit/core/pytorch/graph_substitutions/substitutions/functional_batch_norm.py

```diff
@@ -53,23 +54,22 @@ def get_attributes_from_weights(node: BaseNode) -> Dict:
                         GAMMA: np.ones(node.weights[1].shape),
                         BETA: np.zeros(node.weights[1].shape)}
 
-        has_weight = WEIGHT not in node.framework_attr
-        has_bias = BIAS not in node.framework_attr
+        # Check if weight and/or bias were not given.
+        if KERNEL in node.tensor_input_allocs:
+            weights_dict[GAMMA] = node.weights[node.tensor_input_allocs.index(KERNEL)]
+        elif KERNEL not in node.op_call_kwargs:
+            weights_dict[GAMMA] = node.weights[3]
```

Comment:
same question regarding the arbitrary constant "3"
(also, consider making these constant variables)